File: Sharing-things-backend/Sharing-things-backend/DynamoDB Model/Comment.cs
using Sharing_things_backend.DTOs;

namespace Sharing_things_backend.DynamoDB_Model
{
    public class Comment
    {
        public string Id { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public string Body { get; set; }
        public UserComment Author { get; set; }
    }
}

File: client-app/src/app/models/Comment.ts


export interface VideoCommentsModel{
    videoId:string;
    comments: Comment[];
}

export interface Comment{
    id:string;
    body:string;
    author: CommentAuthor;
    createdAt: string;
}

export interface CommentAuthor{
    username: string;
    displayName: string;
}